openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion/__init__.py

```diff
@@ -74,7 +73,7 @@ class DiscussionXBlock(XBlock, StudioEditableXBlockMixin, XmlParserMixin):  # li
     has_author_view = True  # Tells Studio to use author_view
 
     # support for legacy OLX format - consumed by XmlParserMixin.load_metadata
-    metadata_translations = dict(RawDescriptor.metadata_translations)
+    metadata_translations = dict(XmlParserMixin.metadata_translations)
```

Comment:
`metadata_translations` is specified in only two places: `XModuleDescriptorToXBlockMixin` and `XmlParserMixin`. Both have the same values.